### DISCUSSION TITLE
Oracle feature

### DISCUSSION THREAD

**Discussion by Skimmenthal13** [jchristn/DatabaseWrapper#23] [General]
Hello I have one question about this library.
Why this library doesn't handle also oracle databes? Is It an implementation choice or could be a nice to have this feature?
I'm asking because I found this library very helpful for my projects but I need to handle oracle too.
Eventualy if it is intresting for the comunity I would try to develop this feature.

Thank you so much to share your work!

**Comment by jchristn** [comment]
Hi @Skimmenthal13 thanks for your kind words.  Never had a need to support Oracle, so I never added it.  Cheers
